packages/utils/functions/api/getAllAuthApiErrorMessage/index.ts

```
@@ -0,0 +1,17 @@
+import { get, isArray, isEmpty } from 'lodash'
+
+export const getAllAuthApiErrorMessage = (
+  error: any,
+  { defaultMessage = 'Something went wrong.' } = {},
+) => {
+  let message = defaultMessage
+
+  if (error.response && error.response?.data && typeof error.response?.data === 'object') {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)